ui/pages/remove-snap-account/snap-account-card.tsx

```diff
@@ -24,7 +25,7 @@ export const SnapAccountCard = ({
   const account = mergedAccounts.find(
     (internalAccount: { address: string }) =>
       internalAccount.address === address,
-  );
+  ) as MergedInternalAccount;
```

Comment:
Can we just set the type on the variable declaration like this instead of casting it?

```ts
const account: MergedInternalAccount = mergedAccounts.find(...);
```

Reply:
The `find` method can result in an undefined account thats why i need to cast it, but since this is a remove snap account card, it is always defined.